_posts/2021-04-05-hhvm-4.104.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.104"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.104 is released! This release marks the end of support for 4.98;
+HHVM 4.99&ndash;4.103 remain supported, as do the 4.56, and 4.80 LTS releases.
```

Comment:
If everything would be going according to the old schedule, 4.56 would be dropped now. If 4.56 is going to be kept for longer than anticipated, that might be something worth noting.

Reply:
Good catch, you're right, 4.56 should be dropped now.